cspell.json

```diff
@@ -9,6 +9,7 @@
   ],
   "ignoreRegExpList": [
     "/(```+)[\\s\\S]+?\\1/g",
-    "/`[a-zA-Z-_]+`/g"
+    "/`[a-zA-Z-_]+`/g",
+    "/:[a-z_]+:/g"
```

Comment:
I believe this works for emoji shortcodes but after an (admittedly brief) bit of research I wasn't able to find a formal description of what emoji shortcodes can be. However! If you try `:TADA:` it doesn't work (you get :TADA: instead of :tada:) and some (like :heart_eyes:) have an underscore, so I think this should work. [Hooray for regexes](https://xkcd.com/1171/)!